recipe/meta.yaml

```diff
@@ -1,49 +1,64 @@
 {% set name = "nappy" %}
-{% set version = "1.1.4" %}
-{% set sha256 = "2bc123250e559c574c57ad8d7e4c05814d58f3dc45f70622b1be401d30ec1b20" %}
+{% set version = "2.0.2" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
-  sha256: {{ sha256 }}
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nappy-{{ version }}.tar.gz
+  sha256: b70c562131a470c17246d7b7dcf6c21597e7067534b45d1207e31252c8725926
 
 build:
-  skip: true  # [win or py3k]
-  number: 1001
-  script:
-    - "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
+  entry_points:
+    - na2nc=nappy.script.na2nc:na2nc
+    - nc2na=nappy.script.nc2na:nc2na
+    - nc2csv=nappy.script.nc2csv:nc2csv
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
 
 requirements:
   build:
-    - python
-    - setuptools
   host:
```

Comment:
This is slightly weird because the package imports itself during building so all of `run` is also required in `host`.

Reply:
earlier versions did not have this dependency

Reply:
It comes from here where they import the version: https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=476348&view=logs&j=656edd35-690f-5c53-9ba3-09c10d0bea97&t=e5c8ab1d-8ff9-5cae-b332-e15ae582ed2d&l=137

Reply:
https://github.com/cedadev/nappy/blob/main/nappy/__init__.py